<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartExpWindow2</class>
 <widget class="QWidget" name="StartExpWindow2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>1621</width>
     <height>811</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>1720</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设备信息及状态</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>770</y>
      <width>1601</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="backBtn">
       <property name="text">
        <string>上一步</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextPageBtn">
       <property name="text">
        <string>下一步</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>1601</width>
      <height>661</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QWidget" name="widget_4" native="true">
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>290</y>
          <width>101</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QGroupBox" name="naodian">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>781</width>
          <height>641</height>
         </rect>
        </property>
        <property name="title">
         <string>脑电</string>
        </property>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>14</x>
           <y>16</y>
           <width>761</width>
           <height>621</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="calibraeBtn">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>校准</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="calibraeResult">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 12pt &quot;黑体&quot;;</string>
              </property>
              <property name="text">
               <string>校准结果:暂未校准</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="gazepointWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>581</height>
             </size>
            </property>
            <property name="title">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QWidget" name="cameraWidget" native="true">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>381</width>
              <height>201</height>
             </rect>
            </property>
            <property name="title">
             <string>摄像头画面</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>381</width>
              <height>201</height>
             </rect>
            </property>
            <property name="title">
             <string>心率</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>60</y>
            <width>101</width>
            <height>81</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>781</width>
            <height>211</height>
           </rect>
          </property>
          <property name="title">
           <string>脑电</string>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>11</x>
             <y>11</y>
             <width>761</width>
             <height>191</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="calBtn">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>获取阻抗值</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="finishBtn">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>完成校准</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QTableWidget" name="eegImpTable"/>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>60</y>
            <width>101</width>
            <height>81</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>781</width>
            <height>211</height>
           </rect>
          </property>
          <property name="title">
           <string>皮肤电</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
